config/core/test-configs.yaml

```diff
@@ -2290,6 +2305,7 @@ test_configs:
       - baseline
       - baseline-nfs
       - igt-kms-exynos
+      - kselftest-alsa
```

Comment:
I don't think there are enough XU3 boards online to run any more tests. It's probably better to drop it and rely on other platforms instead.

Reply:
Note that as covered in the commit message the tests are device specific so we can't "rely" on other platforms - there's some shared stuff but a test running on one platform is very much not a direct substitute for a test running on another.

Reply:
Is that tied to the SoC or some audio chip on the XU3?

If this platform is really needed then maybe you could enable some in your lab, or see if another lab wants to add some more.  There might be a couple of spares in Collabora's stock but I can't guarantee they could be added to the lab.

Reply:
The audio subsystem is a combination of both - the current tests mainly cover the off SoC components. It's not *needed* exactly any more than any other testing is needed but it would be good to have the coverage.

I really don't think the current approach is working well for these less generic tests, as someone describing a test for KernelCI this munging of the "what can this board do" and "what do we want to schedule at the current time" into one configuration is confusing to think about (especially when I also had to manually enumerate the tests for my lab) and makes it more complicated to remember what can be enabled where. It feels like we should be making more use of lab filters (especially for the ones where you're asking labs about their capacity), have a separate admin filter or have some way to note that a test is valid for a board even if it's not currently being run.

Reply:
Yes, this is all true. I'm not saying this config schema is ideal, far from it. What we need is people with time and skills to improve the software, so if you want to suggest a better way of doing it or even implement it then please do.  In the meantime, we can only use what is available and it does provide a way to run the things we want to run.

We could easily enough add some blocklist filters in lab-configs to not run some tests on some boards while enabling those tests in test-configs. But then we can't easily tell how test coverage is distributed across different boards. The more dynamic approach I mentioned in another comment is what I would suggest instead, a bit like moving from procedural to functional description of what needs to be tested.

Reply:
We could at least be leaving stuff present but commented out rather than removing lines, that's at least retaining the information about the boards for later.

Reply:
OK: https://staging.kernelci.org/test/plan/id/6259ace1cc7f4bcbf905f4c4/

I suggest we keep this one enabled for now and see next week in production whether it causes any congestion.